.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Just want to double check this was intentional adding this file?

Reply:
It is! Sets up debugging in VSCode.

Reply:
Beautiful and loveley!